### DISCUSSION TITLE
How to Build Infinite Scroll Area with the Scroll Area component

### DISCUSSION THREAD

**Discussion by itzomen** [shadcn-ui/ui#3445] [Q&A]
Hey there, I have been looking into building an infinite scroll area with the `Scroll Area` component but I'm not sure what's the right way to handle scroll events in it, and what would be a recommended approach?

I do not think the `Scroll Area` has any `handleScroll` event on it 🤔 

Any help would be great,
Thanks

**Comment by osman-sultan** [comment]
were you able to get an answer?

**Comment by pablodz** [comment]
x3
